src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java

```diff
@@ -87,4 +69,21 @@ default void doPostCleanup(BibEntry entry) {
             throw new FetcherException("An internal parser error occurred", e);
         }
     }
+
+    /**
```

Comment:
Pleaes move to the original location (maybe reset to the original format and add the newline at line 76 afterwards)